demostore_automation/tests/frontend/test_reg_and_submit.py

```diff
@@ -0,0 +1,23 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+
+driver = webdriver.Chrome()
+driver.get('http://127.0.0.1:8888/Mysite2/')
+
+#finding and clicking on the my account tab
+my_acct = driver.find_element('css selector', '#site-navigation > div:nth-child(2) > ul > li.page_item.page-item-10 > a')
+my_acct.click()
+
+#getting the field email to type in an Email
+u_name = driver.find_element('id', 'reg_email')
+u_name.send_keys('Rasputin@gmail.com')
```

Comment:
If you use the same email then registration will not work more than once.
So generate a new email everytime the script runs.